docs/docs/Deployment/deployment-public-server.md

```diff
@@ -0,0 +1,147 @@
+
+4. Open the URL where ngrok is forwarding your local traffic, such as `https://94b1-76-64-171-14.ngrok-free.app`.
+
+    Your Langflow server is now publicly available at this domain.
+
+## Deploy your MCP server externally
+
+To deploy your Langflow MCP server with ngrok, do the following:
+
+1. Select the Langflow project that contains the flows you want to serve as tools, and then click the **MCP Server** tab.
+
+    Note that the code template now contains your ngrok forwarding address instead of the localhost address.
+    ```json
+    {
+      "mcpServers": {
+        "PROJECT_NAME": {
+          "command": "uvx",
+          "args": [
+            "mcp-proxy",
+            "https://3f7c-73-64-93-151.ngrok-free.app/api/v1/mcp/project/d764c4b8-5cec-4c0f-9de0-4b419b11901a/sse"
+          ]
+        }
+      }
+    }
+    ```
+
+2. Complete the steps in [Connect clients to Langflow's MCP server](/mcp-server#connect-clients-to-use-the-servers-actions) using the ngrok forwarding address.
+
+    For more information, see [MCP server](/mcp-server).
+
+## Serve API requests
+
+Your public Langflow server can serve [Langflow API](/api-reference-api-examples) requests.
+
+To send requests to a public Langflow server's Langflow API endpoints, use the server's domain as the [base URL](/api-reference-api-examples#base-url) for your API requests.
+For example:
+
+```bash
+curl -X POST \
+  "$PUBLIC_SERVER_DOMAIN/api/v1/webhook/$FLOW_ID" \
+  -H "Content-Type: application/json" \
+  -H "x-api-key: $LANGFLOW_API_KEY" \
+  -d '{"data": "example-data"}'
+```
+
+When you create flows in your public Langflow server, the code snippets in the [**API access** pane](/concepts-publish) automatically use your public server's domain.
+
+For example, when used in a script, the following code snippet calls an ngrok domain to trigger the specified flow (`d764c4b8...`):
+
+    ```python
+    import requests
+
+    url = "https://3f7c-73-64-93-151.ngrok-free.app/api/v1/run/d764c4b8-5cec-4c0f-9de0-4b419b11901a"  # The complete API endpoint URL for this flow
+
+    # Request payload configuration
+    payload = {
+        "output_type": "chat",
+        "input_type": "chat",
+        "input_value": "Hello"
+    }
+
+    # Request headers
+    headers = {
+        "Content-Type": "application/json",
+        "x-api-key: $LANGFLOW_API_KEY"
+    }
+
+    try:
+        # Send API request
+        response = requests.request("POST", url, json=payload, headers=headers)
+        response.raise_for_status()  # Raise exception for bad status codes
+
+        # Print response
+        print(response.text)
+
+    except requests.exceptions.RequestException as e:
+        print(f"Error making API request: {e}")
+    except ValueError as e:
+        print(f"Error parsing response: {e}")
+    ```
+
+For a demo of the Langflow API in a script, see the [Quickstart](/get-started-quickstart).
+
+## Share your playground externally
```

Comment:
Just something I noticed: This section is semi-redundant (but more detailed) than this other section on the sharable playground: https://d5rxiv0do0q3v.cloudfront.net/langflow-drafts/docs-quick-guide-for-public-deployment/concepts-publish#shareable-playground

Also, the Embedded chat widget isn't mentioned here.